src/Components/ListItems/Item.js

```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Card, Typography, Button } from "material-ui";
+import { CardMedia, CardContent, CardActions } from "material-ui/Card";
+import { withStyles } from "material-ui/styles";
+
+const styles = {
```

Comment:
Const styles should be imported from the component folder, from a file named styles.css.

Reply:
I agree with you. Just so you know what I was thinking is this:
-When using only a simple style change I would set it Inline.
-If it was more than one type or even more than one style applied to one component I would set a const object.
-If it needed more I would go to a separated file.

The main reason was to better reading of the code and better understanding the code if it where in the same file -- Like in a vue file;